control-plane/plugin/src/resources/node.rs

```diff
@@ -803,15 +701,25 @@ impl Label for Node {
             {
                 Err(source) => match source.status() {
                     Some(StatusCode::UNPROCESSABLE_ENTITY) if output.none() => {
-                        Err(OpError::LabelExists { id: id.to_string() })
+                        Err(OpError::LabelExists {
+                            resource: "Node".to_string(),
+                            id: id.to_string(),
+                        })
                     }
                     Some(StatusCode::PRECONDITION_FAILED) if output.none() => {
-                        Err(OpError::LabelConflict { id: id.to_string() })
+                        Err(OpError::LabelConflict {
+                            resource: "Node".to_string(),
```

Comment:
Shouldn't this be the ResourceKind ?

Reply:
Itna a plugin code. ResourceKind is written in core components